Makefile

```diff
@@ -158,24 +156,26 @@ src/js/generated/_pyodide.out.js:            \
 	cd src/js && npm run build-inner && cd -
 
 dist/pyodide.js:                             \
+		dist/pyodide.asm.js            		 \
```

Comment:
Hmm, I don't think pyodide.asm.js should be a dependency of pyodide.js. What happens if you remove it from the dependency list?

Reply:
@ryanking13 Hi!

Without `dist/pyodide.asm.js` dependency, when build in parallel below error logs occur.

```
Building packages... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0/31   0% Time elapsed: 0:00:04node:events:486
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/src/dist/pyodide.asm.js'
Emitted 'error' event on ReadStream instance at:
    at emitErrorNT (node:internal/streams/destroy:170:8)
    at emitErrorCloseNT (node:internal/streams/destroy:129:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open'
```